src/main.rs

```diff
@@ -142,12 +179,13 @@ fn acceptor() -> Result<TlsAcceptor> {
 struct RequestHandle {
     stream: TlsStream<TcpStream>,
     log_line: String,
+    metadata: Arc<RwLock<FileOptions>>,
```

Comment:
Maybe this should be a `Mutex` instead of `RwLock` because we never use `.read()` access.

Reply:
Ah, yes I was thinking about this but forgot to actually do it. Thanks for reminding!